instrumentation-core/servlet/servlet.gradle

```diff
@@ -1,14 +1,11 @@
-group = 'io.opentelemetry.instrumentation'
-
-apply from: "$rootDir/gradle/java.gradle"
-apply from: "$rootDir/gradle/publish.gradle"
+ext {
+  noShadowPublish = false
```

Comment:
i think the shadow jar is only for use by the corresponding auto-instrumentation modules, so i think we don't want to publish the shadow jar for anything under `:instrumentation-core`
```suggestion
```